scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -43,6 +44,10 @@
 filt = lambda x,filts=args.filterProcs: any([f in x.name for f in filts]) 
 datasets = wremnants.datasets2016.getDatasets(maxFiles=args.maxFiles, filt=filt if args.filterProcs else None)
 
+print('Use v8?', args.v8)
+if args.v8:
+    datasets = wremnants.datasets2016.getDatasets(maxFiles=args.maxFiles, filt=filt if args.filterProcs else None, nanoVersion = "v8")
```

Comment:
I think it would make more sense to use nanoVersion="v8" if args.v8 else "v9" in the argument rather than calling the function twice (won't it trigger some output?)